README.md

```diff
@@ -33,72 +33,33 @@ We recommend [installing GOV.UK Frontend using node package manager
-| Windows          | Edge (latest 2 versions)               | compliant   |
-| Windows          | Google Chrome (latest 2 versions)      | compliant   |
-| Windows          | Mozilla Firefox (latest 2 versions)    | compliant   |
-| macOS            | Safari 9+                              | compliant   |
-| macOS            | Google Chrome (latest 2 versions)      | compliant   |
-| macOS            | Mozilla Firefox (latest 2 versions)    | compliant   |
-| iOS              | Safari for iOS 9.3+                    | compliant   |
-| iOS              | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Samsung Internet (latest 2 versions)   | compliant   |
+- is usable in Internet Explorer 8, 9 and 10
+- lets users adjust or override colours in Windows, Firefox and Chrome
 
-‘Compliant’ means that the components must look as good and function as well as
-they do in other modern browsers.
+## Accessibility
 
-'Functional' means the components may not look perfect in the given browser, but
-must still be usable without errors and without 'looking broken'.
+The GOV.UK Design System team regularly tests GOV.UK Frontend with a range of users to make sure it’s as accessible as possible.
 
-If you are including GOV.UK Frontend as part of a stylesheet that you are
-generating in your application's build pipeline, you will need to [generate and
-include a separate stylesheet in order to support Internet Explorer
-8](docs/installation/supporting-internet-explorer-8.md).
+However, your service will not automatically meet [level AA of WCAG 2.1](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag) if you use GOV.UK Frontend. You must still [do contextual research](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction) to check that your service meets accessibility requirements, especially if you extend or modify components.
 
-[service-manual-browsers]: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in
+You should also use:
 
-## Assistive technology support
+- [the JavaScript from GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
+- [a separate stylesheet](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md) if you support Internet Explorer 8
 
-GOV.UK Frontend will allow you to build services that comply with the [guidance
-in the Service Manual][service-manual-assistive-technologies].
+Your service will not meet level AA of WCAG 2.1 if you use [compatibility mode](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/compatibility.md) to use GOV.UK Frontend with old frameworks or colours.
```

Comment:
I know "old colours" is lifted from the heading of the compatibility mode docs but "old colour palette" would be clearer to me (and we use it too in the docs).

Reply:
Well spotted - looks like I got the style for this wrong on the compatibility page. Will fix!